libs/components/indicators/src/lib/modules/wait/wait.service.spec.ts

```diff
@@ -275,6 +276,205 @@ describe('Wait service', () => {
     }));
   });
 
+  // TODO: Remove when we remove the `Optional` from the dynamic component service in a breaking change
+  describe('basic functionality - without dynamic component service (legacy)', () => {
```

Comment:
These are copies of the existing tests - the only difference is the `beforeEach` which overrides the dynamic service being provided via the root injector.